### PR TITLE
chore: add volunteer sharatkatwa

### PR DESCRIPTION
**Pull request by sharatkatwa**

### CODE REVIEW

File: data/users/Sharat-Katwa.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Sharat Katwa",
+  "bio": "Backend Developer",
+  "image": "https://avatars.githubusercontent.com/u/60929610?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["NodeJS", "ExpressJS","MongoDB", "Javascript"]
```
Comment (codewithvoid) [inline data/users/Sharat-Katwa.json:8]: Move JavaScript to programming

Comment (codewithvoid) [inline data/users/Sharat-Katwa.json:8]: Move MongoDB to `database-development category`

Comment (codewithvoid) [inline data/users/Sharat-Katwa.json:8]: Move MongoDB to `database-development category`

**Comment by codewithvoid** [comment]
Thanks @sharatkatwa for volunteering. Added some small comments.

**Review by sharatkatwa — COMMENTED** [review]
request changes are done

File: data/users/Sharat-Katwa.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (codewithvoid) [inline data/users/Sharat-Katwa.json:1]: File name should be your GitHub username.

`sharatkatwa.json`

**Review by sharatkatwa — COMMENTED** [review]
file name changed

**Comment by codewithvoid** [comment]
Thanks for making the changes. Welcome!

**Pull request merged**